manifests/vhost.pp

```diff
@@ -338,23 +338,24 @@
   }
 
   if $ip {
+    $_ip = any2array($ip)
     if $port {
-      $listen_addr_port = "${ip}:${port}"
-      $nvh_addr_port = "${ip}:${port}"
+      $listen_addr_port = parseyaml(inline_template('<% res="" %><% @_ip.each { |a_ip| res = res + "#{a_ip}:#{@port}," } %><%= res.split(",").to_yaml -%>'))
```

Comment:
That looks better. But this needs stdlib >= 4.2.x. If the Changelog is right this Requirement will jump from 2.4.x to 4.2.x...